webapp/app/static/css/variables.css

```diff
@@ -72,6 +72,15 @@
   --spacing-10: 5.625rem;
   --spacing-11: 7.5rem;
 
+  /* Input Field Widths */
+  --input-width-2: 60px;
+  --input-width-3: 65px;
+  --input-width-4: 85px;
+  --input-width-5: 100px;
+  --input-width-6: 120px;
+  --input-width-10: 280px;
+  --input-width-25: 400px;
```

Comment:
I guess that these have been provided by the design. However, I was wondering why they are not connected to the font by using em?

Reply:
I thought, if we want to make it dependent on the font size, then rather its width than its height. So I thought - as we don't have a monospace font that this is not possible and connecting it to the font's height didn't make sense to me. But I can totally change it if 🎵 you want it that way 🎵 :D